scripts/Diffraction/isis_powder/polaris_routines/polaris_algs.py

```diff
@@ -125,6 +126,18 @@ def generate_ts_pdf(run_number, focus_file_path, merge_banks=False, q_lims=None,
             pdf_output = mantid.Rebin(InputWorkspace=pdf_output, Params=output_binning)
         except RuntimeError:
             return pdf_output
+    # Rename output ws
+    if 'merged_ws' in locals():
+        mantid.RenameWorkspace(InputWorkspace=merged_ws, OutputWorkspace=run_number + '_merged_Q')
+    mantid.RenameWorkspace(InputWorkspace='focused_ws', OutputWorkspace=run_number+'_focused_Q')
+    if isinstance(focused_ws, WorkspaceGroup):
+        for i in range(len(focused_ws)):
+            print(focused_ws[i])
```

Comment:
Do we need this print statement? It results in this output when I run the test scripts:

No loaded focused files found. Searching in output directory...
98533-ResultTOF_1_1
98533-ResultTOF_2_1
98533-ResultTOF_3_1
98533-ResultTOF_4_1
98533-ResultTOF_5_1

First line was always output but seems like we don't need the rest?

Reply:
Oh I put that in for debugging, I must have missed it on cleaning up.